config/vendor-api-experimental.yml

```
@@ -25,6 +25,11 @@ paths:
           default: 1
           minimum: 1
           maximum: 3
+      - name: for_ratified_courses
+        description: Generate applications for courses your organisation awards, not runs. Use a non-empty string value, e.g. 't'
```

Is it worth writing the example here like this to avoid any confusion?
```suggestion
        description: Generate applications for courses your organisation awards, not runs. Use a non-empty string value, e.g. 'for_ratified_courses=true'
```

---

I think it would be clearer to force a value like `true` for this (since we don't yet have any other flag parameters like this to copy). It may be slightly confusing that `for_ratified_courses=false` will activate the setting.

---

Yes, `for_ratified_courses=false` is problematic. I'll change this to require `true`.